h/api_client/subscribers.py

```diff
@@ -0,0 +1,45 @@
+# -*- coding: utf-8 -*-
+import pyramid.events
+
+import h.api_client.api_client as api_client
+
+
+def _get_api_url(request):
+    """Return the configured base URL for this app's Hypothesis API instance.
+
+    Returns the value of the h.api_url setting from the config file,
+    or if that doesn't exist the "/api" route from the WSGI app (which assumes
+    that we're running the app with the "api" feature enabled).
+
+    Always returns the URL _without_ a trailing /, despite what the user might
+    have entered in the config file.
+
+    """
+    return request.registry.settings.get(
+        "h.api_url", request.resource_url(request.root, "api")).rstrip("/")
```

Comment:
My suggestion might be to add an `includeme(config)` function to `h/api/__init__.py`. Have it normalize this setting and then `config.include('.db')` and `config.include('.views')`. Then, change `h/app.py#create_api` to include `api` rather than each part of it separately.